crates/shared/src/consts.rs

```diff
@@ -1 +1,2 @@
 pub const EXPECTED_RPC_VERSION: &str = "0.7.0";
+pub const RPC_URL_VERSION: &str = "v0_7";
```

Comment:
this may be a nitpick, but I suppose this will prevent users from using older rpcs for tests? are we 100% sure we want to go that way?

Reply:
Don't we update the tests to use the latest possible version in foundry? Like [here](https://github.com/foundry-rs/starknet-foundry/commit/b458b753e9e6e2f69703da0aafc917230c2b237a#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR20)

Reply:
yup, newer versions of snfoundry suoport newer versions of rpc, but I don't think they block the older ones. So technically it should be possible to use newest snforge (say with rpc 0.7 support) and run it against rpc 0.6? I assume that, becase you raise a warning about rpc version mismatch, and not an error. or is that not the case at all?

Reply:
Yes, it emits a warning when an older version is used. I don't want to add another env variable for setting rpc version, though.

Reply:
An alternative would be to just have a NODE_URL variable, that holds complete url right? you wouldn't have to set specific version in code, you could just change it in .env file/in workflow file in CI. But it all probably depends on your (snforge) needs etc. @Arcticae any thoughts on that?

Reply:
Isn't this only for tests anyway? I guess if we want to run the tests against multiple ones we'd have to parametrise our tests somehow but i think it's out of scope here

Reply:
ok, let's leave this as is for now then :+1: